docs/_documentations/troubleshooting.md

```diff
@@ -231,3 +232,69 @@ Issue link:
 If you turn off `auto build` for a Node.js project when you run Codewind locally, it has no effect. Changes you make to your code automatically start or restart a build even though `auto build` is disabled.
 
 ***
+# Appsody with Codewind
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## Node.js and Swift templates remain in the starting state
+Node.js and Swift templates remain in the starting state. These projects don't start a server for Codewind to ping.
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## A project build error appears after you create an initial project
+If you use Eclipse and either Java MicroProfile or Spring Appsody templates, you might receive a `Project build error: Non-resolvable parent POM` error after you create the initial project.
+
+**Workaround** Complete the following instructions to work around the error:
+1. Right-click the project and select **Show Log Files**>**Show All**.
+2. If your `.m2` cache is empty, or if you have not previously created a Java Appsody project, the dependencies will download, and the `[Container] Installing parent dev.appsody` message appears.
+3. Wait until the cache completes. You can wait until the Project status is `Running`, or, if you use the MicroProfile template, you can wait until the `Liberty defaultServer` starts.
+4. Right-click the **Project** from the Project Explorer and select **Maven**>**Update Project...**.
+5. Accept the defaults and click **OK**. The project is configured, and the `Project build error: Non-resolvable parent POM` disappears.
+
+After you create the initial project and set the `.m2` cache, new projects begin to be configured properly.
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## An Unknown error appears on line one of the pom.xml file
+If you use an Eclipse IDE for Enterprise Developer EPP prior to version 2019.06, you might see an `Unknown` validation error in the `pom.xml` file.
+
+**Workaround** Switch to version 2019.06 or later, or see (Cannot import any project into Eclipse with maven-jar-plugin 3.1.2)[https://bugs.eclipse.org/bugs/show_bug.cgi?id=547340].
```

Comment:
The link syntax was swapped, it should be `[Cannot import any project into Eclipse with maven-jar-plugin 3.1.2](https://bugs.eclipse.org/bugs/show_bug.cgi?id=547340)`